code/game/gamemodes/scoreboard.dm

```diff
@@ -275,15 +334,21 @@
 			break // Should only need one go-round, probably
 		var/nukedpenalty = 0
 		for(var/obj/machinery/nuclearbomb/NUKE in machines)
-			if (NUKE.r_code == "Nope") continue
+			if(NUKE.r_code == "Nope")
+				continue
 			var/turf/T = NUKE.loc
 			bombdat = T.loc
-			if (istype(T,/area/syndicate_station) || istype(T,/area/wizard_station) || istype(T,/area/solar/) || istype(T,/area)) nukedpenalty = 1000
-			else if (istype(T,/area/security/main) || istype(T,/area/security/brig) || istype(T,/area/security/armory) || istype(T,/area/security/checkpoint2)) nukedpenalty = 50000
-			else if (istype(T,/area/engine)) nukedpenalty = 100000
-			else nukedpenalty = 10000
+			if(istype(T,/area/syndicate_station) || istype(T,/area/wizard_station) || istype(T,/area/solar/) || istype(T,/area))
```

Comment:
WAIT A SECOND DYLAN.

You're doing ISTYPES FOR AREAS ON TURFS?

THAT'S ALWAYS FALSE.